tm2/pkg/sdk/baseapp.go

```diff
@@ -389,10 +392,6 @@ func (app *BaseApp) Query(req abci.RequestQuery) (res abci.ResponseQuery) {
 	default:
 		return handleQueryCustom(app, path, req)
 	}
-
```

Comment:
Why remove this error wrap?

Reply:
I removed it because it was unreacahble. I'll add the error message back in proper position